modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalFolderServiceImpl.java

```diff
@@ -130,6 +130,21 @@ public JournalFolder getFolder(long folderId) throws PortalException {
 		return folder;
 	}
 
+	@Override
+	public JournalFolder getFolderByExternalReferenceCode(
+			long siteId, String externalReferenceCode)
```

Comment:
To mantain the same notation, we should rename `siteId` for `groupId` as in the rest of the methods